unit_tests/bg/search_url/ref/thomas_moffat_29_october_new_style.json

```diff
@@ -1,5 +1,5 @@
 {
   "std": {
-    "url": "https://billiongraves.com/search/results?given_names=Thomas&family_names=Moffat&cemetery_state=Scotland&cemetery_country=United%20Kingdom&year_range=5&size=15"
+    "url": "https://billiongraves.com/search/results?CollectionID=&CatalogID=&PageNumber=1&PageSize=20&EventBirthYearRange=5&EventDeathYearRange=5"
```

Comment:
Seems like this has less information than before?

Reply:
Due to failure to gather cemetery data. Now fixed.